include/dirac_quda.h

```diff
@@ -521,6 +528,8 @@ namespace quda {
   class DiracWilson : public Dirac {
 
   protected:
+    double distance_pc_alpha0;
+    int distance_pc_t0;
```

Comment:
Should both `distance_pc_t0` and `distance_pc_alpha0` be members of `Dirac` and not `DiracWilson`.  In the future we want want distance preconditioning in other Dirac operators, so this would be more forward looking,

Reply:
The two parameters are moved to `Dirac`.